json/src/main/scala/com/gu/contentapi/json/CirceDecoders.scala

```diff
@@ -95,6 +70,18 @@ object CirceSerialization {
   implicit val mostViewedVideoDecoder = Decoder[MostViewedVideo]
   implicit val networkFrontDecoder = Decoder[NetworkFront]
   implicit val packageDecoder = Decoder[Package]
+  implicit val itemResponseDecoder = Decoder[ItemResponse]
+  implicit val searchResponseDecoder = Decoder[SearchResponse]
+  implicit val editionsResponseDecoder = Decoder[EditionsResponse]
+  implicit val tagsResponseDecoder = Decoder[TagsResponse]
+  implicit val sectionsResponseDecoder = Decoder[SectionsResponse]
+  implicit val atomsResponseDecoder = Decoder[AtomsResponse]
+  implicit val packagesResponseDecoder = Decoder[PackagesResponse]
+  implicit val errorResponseDecoder = Decoder[ErrorResponse]
+  implicit val videoStatsResponseDecoder = Decoder[VideoStatsResponse]
+  implicit val atomsUsageResponseDecoder = Decoder[AtomUsageResponse]
+  implicit val removedContentResponseDecoder = Decoder[RemovedContentResponse]
+  implicit val storyPackageArticleDecoder = Decoder[com.gu.storypackage.model.v1.Article]
```

Comment:
Seems weird that we need this one, especially as we already have a `Decoder[Package]` above it on line 72